blueprints/node-webkit/files/nw-runner.js

```diff
@@ -0,0 +1,35 @@
+var exec = require('child_process').exec;
+var NW_VERSION = '0.12.0';
+var os = require('os');
+var fs = require('fs');
+var findNw = require('./node_modules/ember-cli-node-webkit/lib/helpers/find-nw');
```

Comment:
No, because nw-runner.js is running in the context of the plugin. so it would require the application consuming the plugin to take a dependency on `find-nw`. I don't think they should have to do that.